internal/helm/chart/builder_local.go

```diff
@@ -142,7 +142,7 @@ func (b *localChartBuilder) Build(ctx context.Context, ref Reference, p string,
 	// Merge chart values, if instructed
 	var mergedValues map[string]interface{}
 	if len(opts.GetValuesFiles()) > 0 {
-		if mergedValues, err = mergeFileValues(localRef.WorkDir, opts.ValuesFiles); err != nil {
+		if mergedValues, err = mergeFileValues(localRef.WorkDir, opts.ValuesFiles, opts.IgnoreMissingValuesFiles); err != nil {
```

Comment:
Later on, there is a call to `OverwriteChartDefaultValues(loadedChart, mergedValues)`. If this is successful we append the `build` result with the `ValuesFiles` in `result.ValuesFiles = opts.GetValuesFiles()`. This is later used in `Summary` to report in the CR conditions the build result. If some files are missing, the reported merged files will be erronous.

Reply:
So, would you like me to adjust the `mergeFileValues` func to calculate and return the *actual* list of merged values files, and then use the amended `valuesFiles` for the `result.ValuesFiles = opts.GetValuesFiles()` assignment below? Is that enough?